test/integration/associations/belongs-to-many.test.js

```diff
@@ -1417,7 +1417,7 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), () => {
 
   describe('hasAssociations with binary key', () => {
     beforeEach(function() {
-      const keyDataType = ['mysql', 'mariadb', 'db2'].includes(dialect) ? 'BINARY(255)' : DataTypes.BLOB('tiny');
+      const keyDataType = ['mysql', 'mariadb', 'db2'].includes(dialect) ? 'BINARY(255)' : dialect === 'oracle' ? DataTypes.STRING(255, true) : DataTypes.BLOB('tiny');
```

Comment:
Doesn't oracle support the `BINARY()` data type? :o

Reply:
No, Oracle DB doesn't have a Binary type

Reply:
Should we use this instead?
```
let keyDataType = DataTypes.BLOB('tiny');
 if (['mysql', 'mariadb', 'db2'].includes(dialect)) {
    keyDataType =  'BINARY(255)';
} else if (dialect === 'oracle') {
    keyDataType =  DataTypes.STRING(255, true);
}
```